changelog/_unreleased/2025-03-31-less-db-calls-for-getavailablesortings.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Fewer DB calls for getAvailableSortings
+author: Björn Meyer
+author_email: b.meyer@shopware.com
+author_github: @BrocksiNet
+---
+# Core
+* Changed `src/Core/Content/Product/Cms/ProductListingCmsElementResolver.php` to load all active sorting options once from the db and reuse them later.
```

Comment:
```suggestion
* Changed `src/Core/Content/Product/Cms/ProductListingCmsElementResolver.php` to load all active sorting options once from the DB and reuse them later.
```

I also like to use the class name instead of the filename :thinking: